project-base/storefront/components/Blocks/Product/SliderProductItem.tsx

```diff
@@ -75,6 +85,18 @@ export const SliderProductItem: FC<SliderProductItemProps> = ({
                         </div>
                     </a>
                 </ExtendedNextLink>
+                <div className="mb-2 flex justify-end gap-2">
+                    <ProductCompareButton
+                        isMainVariant={product.isMainVariant}
+                        isProductInComparison={isProductInComparison}
+                        onProductInComparisonClick={onProductInComparisonClick}
+                    />
+                    <ProductWishlistButton
+                        isMainVariant={product.isMainVariant}
+                        handleProductInWishlist={handleProductInWishlist}
+                        isInWishlist={isInWishlist}
+                    />
```

Comment:
Ugh, the issue with naming goes even deeper :laughing:

Reply:
renamed everywhere – 8fdca51562